npm install fails, webdriver-manager update command not found

**Do you want to request a *feature* or report a *bug*?**
npm install fails when running webdriver-manager update


**What is the current behavior?**


AlexandrusMBP3:WyliodrinSTUDIO alexandru$ npm install angular-ui-tree

> angular-ui-tree@2.22.4 postinstall /Users/alexandru/programe/WyliodrinSTUDIO/node_modules/angular-ui-tree
> webdriver-manager update

sh: webdriver-manager: command not found
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "angular-ui-tree"
npm ERR! node v6.9.5
npm ERR! npm  v3.10.10
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn

npm ERR! angular-ui-tree@2.22.4 postinstall: `webdriver-manager update`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the angular-ui-tree@2.22.4 postinstall script 'webdriver-manager update'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the angular-ui-tree package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     webdriver-manager update
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs angular-ui-tree
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls angular-ui-tree
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/alexandru/programe/WyliodrinSTUDIO/npm-debug.log
AlexandrusMBP3:WyliodrinSTUDIO alexandru$ 

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via https://plnkr.co or similar.**

npm install angular-ui-tree

--- a/package.json
+++ b/package.json
@@ -42,7 +42,7 @@
     "node": ">=6"
   },
   "scripts": {
-    "postinstall": "webdriver-manager update",
+    "pretest:e2e": "webdriver-manager update",
     "test:e2e": "protractor",
     "test": "gulp build && npm run test:e2e",
     "coveralls": "cat ./coverage/lcov/lcov.info | coveralls",

